ui/src/components/filters/QasFilters.vue

```diff
@@ -150,7 +162,7 @@ export default {
     },
 
     debounce () {
-      return this.useSearchOnType ? '500' : ''
+      return this.useSearchOnType ? '800' : ''
```

Comment:
`return this.useSearchOnType && '800'` resolve ou precisa retornar uma string?

Reply:
Não, essa prop precisa ser string ou number, do jeito que falou pode ficar um boolean.